Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj

```diff
@@ -36,13 +36,15 @@
     </When>
     <Otherwise>
       <ItemGroup>
-        <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="7.3.67" />
-        <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="7.3.30" />
+        <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="7.3.168" />
+        <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="7.3.160" />
       </ItemGroup>
     </Otherwise>
   </Choose>
   <ItemGroup>
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="9.0.0" />
+    <PackageReference Include="dbup-postgresql" Version="5.0.40" />
```

Comment:
DbUp has been introduced to handle database migration scripts at startup (for the EdFi_Security_Test database) and at first attempt to connect to each ODS (supporting multi-tenant configurations).